### DISCUSSION TITLE
File/Folder uploading - HFS 3

### DISCUSSION THREAD

**Discussion by redtrillix** [rejetto/hfs#61] [Q&A] [answered]
Will we have upload support with HFS 3 or if we do, where/how would I enable it?
Cause I found that really useful in HFS 2, and that's one of the reasons why I haven't switched yet.

Thanks in advance.

**Comment by rejetto** [comment, marked as answer]
it will, have a look https://github.com/rejetto/hfs/blob/main/todo.md

**Reply by redtrillix** [reply]
Thank you very much!
